### ISSUE TITLE
/Filter/Fl 

### ISSUE THREAD

**Issue by pts** [pts/pdfsizeopt#23]
pdfizeopt doesn't recognize `/Filter/Fl` in `<</Filter/Fl/First 24/Length 102/N 4/Type/ObjStm>>stream`. It should recognize it as `/Filter/FlateDecode`. Also `/Filter/A85` ahould be recognized ad `/Filter/ASCII85Decode`.

There are some PDFs in the wild which have abbreviated filter names. They are not valid PDF, so pdfsizeopt should unabbreviate the filter names.

```
% Contrary to the published PDF (1.3) specification, Acrobat Reader
% accepts abbreviated filter names everywhere, not just for in-line images,
% and some applications (notably htmldoc) rely on this.
/unabbrevfilterdict mark
  /AHx /ASCIIHexDecode  /A85 /ASCII85Decode  /CCF /CCITTFaxDecode
  /DCT /DCTDecode  /Fl /FlateDecode  /LZW /LZWDecode  /RL /RunLengthDecode
```

**Comment by pts** [comment]
Fixed in f77bf784e5a22f4804218ecdf6886ba5ec5e256e.

**Comment by rbrito** [comment]
Thanks a lot for making the generated PDF files (a bit) more compliant!
